modules/lambdas/main.tf

```diff
@@ -1,83 +1,154 @@
+  region  = var.region
+  profile = var.aws_profile
+}
 
+
+## Local Variables
 locals {
-  default_tags = {
-    Deployment = var.prefix
-  }
+  tags         = merge(var.tags, { Deployment = var.prefix })
+  orca_buckets = [for k, v in var.buckets : v.name if v.type == "orca"]
 }
 
+
+## Referenced Modules
+# lambda_security_group - Security Groups module reference
+# ------------------------------------------------------------------------------
 module "lambda_security_group" {
   source = "../security_groups"
-  tags = var.tags
-  vpc_id = var.vpc_id
-  prefix = var.prefix
+  ## --------------------------
+  ## Cumulus Variables
+  ## --------------------------
+  ## REQUIRED
+  aws_profile = var.aws_profile
+  prefix      = var.prefix
+  vpc_id      = var.vpc_id
+  ## OPTIONAL
+  region = var.region
+  tags   = local.tags
+  ## --------------------------
+  ## ORCA Variables
+  ## --------------------------
+  ## OPTIONAL
+  database_port = var.database_port
 }
 
+# restore_object_arn - IAM module reference
+# ------------------------------------------------------------------------------
 module "restore_object_arn" {
   source = "../iam"
-  buckets = var.buckets
-  prefix = var.prefix
+  ## --------------------------
+  ## Cumulus Variables
+  ## --------------------------
+  ## REQUIRED
+  aws_profile              = var.aws_profile
+  buckets                  = var.buckets
   permissions_boundary_arn = var.permissions_boundary_arn
+  prefix                   = var.prefix
+  ## OPTIONAL
+  region = var.region
+  tags   = local.tags
+  ## --------------------------
+  ## ORCA Variables
+  ## --------------------------
+  ## OPTIONAL
+  orca_recovery_buckets = var.orca_recovery_buckets
 }
 
-// You don't need to specify orca version if you used source_code_hash
-resource "aws_lambda_function" "db_deploy" {
-  filename      = "${path.module}/../../tasks/db_deploy/db_deploy.zip"
-  function_name = "${var.prefix}_db_deploy"
-  source_code_hash = filemd5("${path.module}/../../tasks/db_deploy/db_deploy.zip")
+
+## =============================================================================
+## Ingest Lambdas Definitions and Resources
+## =============================================================================
+
+# copy_to_glacier - Copies files to the ORCA S3 Glacier bucket
+# ==============================================================================
+resource "aws_lambda_function" "copy_to_glacier" {
+  ## REQUIRED
+  function_name = "${var.prefix}_copy_to_glacier"
   role          = module.restore_object_arn.restore_object_role_arn
-  handler       = "db_deploy.handler"
-  runtime       = "python3.7"
-  timeout       = var.lambda_timeout
-  description   = "Deploys the Disaster Recovery database"
+
+  ## OPTIONAL
+  description      = "ORCA archiving lambda used to copy data to an ORCA S3 glacier bucket."
+  filename         = "${path.module}/../../tasks/copy_to_glacier/copy_to_glacier.zip"
+  handler          = "handler.handler"
```

Comment:
Should this be changed to "copy_to_glacier.handler"  in line 85?
The handler.py should also be changed to copy_to_glacier.py in that case.

Reply:
No, handler.handler is correct in this context. If you look at the task https://github.com/nasa/cumulus-orca/tree/master/tasks/copy_to_glacier you will notice that the main python script is called `handler.py` and the function inside being called is `handler`. That is where the handler is being called `handler.handler`.

I agree that we should change it to better match convention from `handler.py` to `copy_to_glacier.py`. I will make those changes shortly.

Reply:
Changes to lambda have been made. please review them for any additional issues.

Reply:
looks good to me.